modules/animate/main/index.coffee

```diff
@@ -1,8 +1,13 @@
-### istanbul ignore next: ignore coffeescript generated code that can't be covered  ###
+transition = require('modules/transition/main')
+EventEmitter = require('modules/event-emitter/main')
+interpolate = require('modules/interpolate/main')
+select = require('modules/selection/main')
+hxMap = require('modules/map/main')
```

Comment:
Does this need to be `hxMap` because there's a global `map` or is it so it's obvious that it's not `Array::map`?